lib/email/utils/helpers.js

```diff
@@ -45,7 +47,22 @@ function logFlowEventFromMessage(log, message, type) {
   }
 }
 
+function getPopularDomain(email) {
```

Comment:
tiny naming nit: I wonder if something like `getAnonymizedEmailDomain` might make it more clear *why* this function does what it does.  The calling code doesn't care about "popular", it cares about not logging PII.